.gitignore

```diff
@@ -44,6 +44,9 @@ coverage.xml
 .project
 .pydevproject
 
+# Vim
+*.swp
```

Comment:
Maybe we should add this everywhere?  Or alternatively, you can add it to your .gitignore_global.  That’s what I did with .vscode:

[core]
    pager = less -F
    excludesfile = /Users/zephyr/.gitignore_global
$ cat ~/.gitignore_global
.python-version
.vscode

Then we don't have to have everyone's editors in every repo.

Reply:
I took the fact that several things we don't use are already there as indication it would be okay to expand on, but I'm fine removing it if this policy we don't want to set a precedent for.

Reply:
I'm open to it but only if we discuss it as a group first.  I'd like to have a clear global policy.